src/aw-client.ts

```diff
@@ -24,14 +24,24 @@ export interface AppEditorActivityEvent extends AppEditorActivityHeartbeat {
     duration: number;
 }
 
+export interface Bucket {
+  id: string,
+  name: string,
+  type: string,
+  client: string,
+  hostname: string,
+  created: Date
+  last_update?: Date
+}
+
 interface HeartbeatQueueItem {
-    onSuccess: Function;
-    onError: Function;
-    pulsetime: number;
+    onSuccess: (heartbeat: Heartbeat) => void;
+    onError: (err: AxiosError) => void;
+    pulseTime: number;
```

Comment:
We don't camelCase `pulsetime` anywhere else, so probably shouldn't here either for consistency.